api/nakadi-event-bus-api.yaml

```diff
@@ -920,6 +920,12 @@ definitions:
               from the header into the metadata by Nakadi to avoid clients having to explicitly copy this.
             type: string
             example: 'JAh6xH4OQhCJ9PutIV_RYw'
+          partition:
```

Comment:
@jkliff are you going to make any further changes to this PR? Otherwise I would merge it in order to avoid conflicts, since I'm also changing the event publishing controller.